runtime/src/accounts_db.rs

```diff
@@ -145,6 +145,17 @@ pub enum StoreReclaims {
     Ignore,
 }
 
+/// specifies how to return zero lamport accounts
+/// This will only be useful until a feature activation occurs.
+#[derive(Clone, Copy)]
+pub enum LoadZeroLamports {
+    /// return None if loaded account has zero lamports
+    None,
```

Comment:
`None` is in rust prelude. Redefining it to mean something else can be confusing.

This whole `enum LoadZeroLamports` imo is unnecessary and more confusing than just passing around:
```rust
return_none_for_zero_lamport_accounts: bool,
```
across the callstack.

Reply:
> `None` is in rust prelude.

I acknowledge your concern about `None`. But it will always be prefixed like `LoadZeroLamports::None`. Feel free to suggest an alternative.

Reply:
> This whole `enum LoadZeroLamports` imo is unnecessary

Passing an additional `true`/`false` I consider to be potentially confusing in this case. The intention here is to make this explicit until the feature activation occurs. Then, this enum and the parameters will all disappear.